LoggingHelper.cpp

```diff
@@ -0,0 +1,143 @@
+#include "pch.h"
+#include "LoggingHelper.h"
+
+#include <fstream>
+
+using namespace NppShell::Helpers;
+
+LoggingHelper::LoggingHelper()
+{
+    appDataLoggingEnabled = IsAppDataLoggingEnabled();
+
+    if (!appDataLoggingEnabled)
+    {
+        return;
+    }
+
+    const wstring logFileFolder = CreateAppDataFolder();
+    logFilePath = logFileFolder + L"\\NppShell.txt";
```

Comment:
Is it better to use **NppShell.log** instead of **NppShell.txt** ?

Reply:
Yep, I have changed it 🙂